### PR TITLE
July release notes

### PR DESCRIPTION
**Pull request by NWeber-SL**

### CODE REVIEW

File: release-notes/july-2019.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# ShiftLeft July 2019 Updates
+
+## ShiftLeft Ocular
+
+* **[Scala Support (Early Access)](../introduction/requirements.md)**. You can now use ShiftLeft Ocular to examine your Scala applications for vulnerabilities and data leakage. As part of this early access, Scala support is available for version 2.12+.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/july-2019.md]:
```suggestion
* **[Scala Support (Early Access)](../introduction/requirements.md)**. You can now use ShiftLeft Ocular to examine your Scala applications for vulnerabilities and data leakage. As part of this early access, Scala support is available for Ocular version 2.12+.
```

File: release-notes/july-2019.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+* **[Ability to Automatically Provision Your Servers](../using-ocular/about/ocular-memory-size.md)**. You can now automate the provision of your servers to ensure that you have the required amount of memory to run ShiftLeft Ocular without impacting performance.
+      
+## ShiftLeft Inspect
+
+* **[Scala Support (Early Access)]](../introduction/requirements.md)**. You can now use ShiftLeft Inspect to analyze your Scala applications for vulnerabilities and data leakage. As part of this early access, Scala support is available for version 2.12+.
```
Comment (vlad-alexandru-ionescu) [inline release-notes/july-2019.md]: The version number doesn't make sense in this case - being a cloud-based offering, Inspect does not have a specific version. It just gets continuous updates.

File: release-notes/july-2019.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+* **[Analysis Results Notification](../using-inspect-protect/inspect/analysis-results.md)**. The first time you submit an application, or if your application has major library changes, it may take ShiftLeft Inspect a day or two to complete a full analysis. In those situations, ShiftLeft Inspect initially displays the partial analysis results, which is indicated by a message in the UI. From this message, you can specify that you want to receive notification when the full analysis results are available.
+
+* **Vulnerability Detection Improvements**. The detection rules for vulnerabilities have been improved. Rules that resulted in too many or unclear results have been removed, especially around handling sensitive data storage and cache handling.`
```
Comment (vlad-alexandru-ionescu) [inline release-notes/july-2019.md]:
```suggestion
* **Vulnerability Detection Improvements**. The detection rules for vulnerabilities have been improved. Rules that resulted in too many or unclear results have been removed, especially around handling sensitive data storage and cache handling.
```

File: release-notes/july-2019.md (hunk 1 above)
Comment (NWeber-SL) [reply release-notes/july-2019.md]: Fixed

File: release-notes/july-2019.md (hunk 2 above)
Comment (NWeber-SL) [reply release-notes/july-2019.md]: Version number removed

File: release-notes/july-2019.md (hunk 3 above)
Comment (NWeber-SL) [reply release-notes/july-2019.md]: Fixed

File: release-notes/july-2019.md (hunk 1 above)
Comment (vlad-alexandru-ionescu) [reply release-notes/july-2019.md]: I misread this - I think you had it correct the first time. I thought this is about the Ocular version - but it seems 2.12+ is the Scala version supported. Mind reverting to your initial version?

Comment (NWeber-SL) [reply release-notes/july-2019.md]: Reverted.

**Pull request merged**